network/socket.js

```diff
@@ -89,4 +88,14 @@ module.exports = function Socket({
       callback();
     });
   }
+}
+
+function validate({ name, value, type }) {
+  if (value === undefined || value === null) throw new TypeError(`${name} is missing`);
+  // eslint-disable-next-line valid-typeof
```

Comment:
what's wrong with this one ?

Reply:
this rule requires a comparison to a literal string, Since I'm comparing to a parameter it fails the lint. In this specific case I want to use the parameter (thats the point of this function)